src/type.d.ts

```diff
@@ -0,0 +1,24 @@
+type VideoStatus = {
+  paused: boolean;
+  startTime?: Time;
+};
+
+type BrowserMessage = {
```

Comment:
A runtime event should have prefix to avoid conflicting with other web pages/extensions. For example `ticklabvn.tpulse.NEW_VIDEO`.

Reply:
Already updated